pkg/internal/apis/certmanager/validation/certificate_for_issuer.go

```diff
@@ -48,7 +48,7 @@ func ValidateCertificateForACMEIssuer(crt *cmapi.CertificateSpec, issuer *cmapi.
 	el := field.ErrorList{}
 
 	if crt.IsCA {
-		el = append(el, field.Invalid(specPath.Child("isCA"), crt.KeyAlgorithm, "ACME does not support CA certificates"))
+		el = append(el, field.Invalid(specPath.Child("isCA"), crt.IsCA, "ACME does not support CA certificates"))
```

Comment:
Oh, was that a bug in the validation?

Reply:
Yep 🤦 but it isn't an actual validation bug, rather the message logged to the user would be wrong